app/src/main/java/androidPracticals/InheritanceClass.kt

```diff
@@ -0,0 +1,22 @@
+package androidPracticals
+
+open class BaseClass {
+    val name = "Dhyana"
+    fun printFunction() {
+        println("Base Class")
+    }
+}
+
+class DerivedClass: BaseClass() {
+    fun print() {
```

Comment:
print is not valid name

Reply:
and what is the use of inheritance when we are making different function in child class